subprojects/provider-plugins/src/main/kotlin/org/gradle/kotlin/dsl/provider/plugins/DefaultProjectSchemaProvider.kt

```diff
@@ -87,6 +88,7 @@ fun targetSchemaFor(target: Any, targetType: TypeOf<*>): TargetTypedSchema {
             sourceSetsOf(target)?.forEach { sourceSet ->
                 collectSchemaOf(sourceSet, typeOfSourceSet)
             }
+            collectSchemaOf(target.dependencies, typeOfDependencyHandler)
```

Comment:
Could you please move that line above the warning about source set? 5 lines up

Reply:
Sure.

Reply:
Done. 👍

Reply:
Wow, I totally misread that as:
> Could you please _**remove**_ that line above the warning about source set? 5 lines up

My mistake, sorry about that.